docs/bin/testing_formatter.sh

```diff
@@ -20,7 +20,12 @@ $(for test in $(../../bin/ansible-test sanity --list-tests --allow-disabled); do
 
 EOF
 
+SHA_CMD="sha1sum"
+if ! which ${SHA_CMD} > /dev/null 2>&1; then
+    SHA_CMD="sha1"
```

Comment:
The command on macOS is `shasum`. I'm not sure we want to change this to a `case` statement or add another conditional to get the correct command on macOS.

Reply:
I am not sure how we might be able to avoid an extra conditional here as each platform is a bit different. Any suggestions for improvement?

Reply:
I was thinking a `case` statement like this:
```shell
DIST=$(uname)

case $DIST in
    OpenBSD )
        # something
        ;;
    Darwin )
        # something
        ;;
    Linux )
        # something
        ;;
    * )
        # default
        ;;
esac
```

Or an extra conditional would be fine too. Whatever is simplest.